runtime/doc/autocmd.txt

```diff
@@ -1,4 +1,4 @@
-*autocmd.txt*   For Vim version 7.4.  Last change: 2015 Jul 10
+*autocmd.txt*   For Vim version 7.4.  Last change: 2015 Jul 20
```

Comment:
@watiko I assume you did not include this change:

```
Usage example: Check for the existence of the
				directory in the 'backupdir' and 'undodir'
```

because we already autocreate those dirs?